### PR TITLE
Fix Ubuntu/Debian setup when $PATH contains spaces

### PR DESCRIPTION
**Pull request by neon4272**
## Description

The `echo` command in the note doesn't work on Ubuntu/Debian WSL. An `export` command is simpler and does work on WSL systems.
 
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [x] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by fauxpark** [comment]
This only works as long as the terminal stays open, it's not persistent - which is why we need to write to .bashrc.

**Comment by spidey3** [comment]
The export is different in function though, in that it is not persistent. 

I am also surprised that the echo command does not work for you. What error message do you receive?

**Comment by neon4272** [comment]
```
-bash: /home/neon4272/.bashrc: line 118: syntax error near unexpected token `('
-bash: /home/neon4272/.bashrc: line 118: `PATH=/home/neon4272/.local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/mnt/c/Program Files/WindowsApps/CanonicalGroupLimited.Ubuntu20.04onWindows_2004.2020.424.0_x64__79rhkp1fndgsc:/mnt/c/Program Files/Python38/Scripts/:/mnt/c/Program Files/Python38/:/mnt/c/Program Files (x86)/Common Files/Intel/Shared Libraries/redist/intel64/compiler:/mnt/c/Program Files (x86)/Common Files/Oracle/Java/javapath:/mnt/c/Windows/system32:/mnt/c/Windows:/mnt/c/Windows/System32/Wbem:/mnt/c/Windows/System32/WindowsPowerShell/v1.0/:/mnt/c/Windows/System32/OpenSSH/:/mnt/c/Program Files/nodejs/:/mnt/c/Program Files/ffmpeg/bin:/mnt/c/Program Files/dotnet/:/mnt/c/MinGW/bin:/mnt/c/Program Files (x86)/Brackets/command:/mnt/c/Program Files/Git/cmd:/mnt/c/Users/feath/AppData/Local/Programs/Python/Python38/Scripts/:/mnt/c/Users/feath/AppData/Local/Programs/Python/Python38/:/mnt/c/Users/feath/AppData/Local/Microsoft/WindowsApps:/mnt/c/Program Files/heroku/bin:/mnt/c/Users/feath/AppData/Local/atom-beta/bin:/mnt/c/Users/feath/AppData/Roaming/npm:/mnt/c/Users/feath/AppData/Local/GitHubDesktop/bin:/snap/bin'
```
It appears that bash thinks `Program Files (x86)` is a pipe. This results in a syntax error.
As for `export` not being persistent, I think adding the `export` command as another note under the `echo` command would make sense

File: docs/newbs_getting_started.md [hunk 1]
```diff
@@ -98,7 +98,7 @@ It's possible, that you will get an error saying something like: `bash: qmk: com
 This is due to a [bug](https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=839155) Debian introduced with their Bash 4.4 release, which removed `$HOME/.local/bin` from the PATH. This bug was later fixed on Debian and Ubuntu.
 Sadly, Ubuntu reitroduced this bug and is [yet to fix it](https://bugs.launchpad.net/ubuntu/+source/bash/+bug/1588562).
 Luckily, the fix is easy. Run this as your user: `echo "PATH=$HOME/.local/bin:$PATH" >> $HOME/.bashrc && source $HOME/.bashrc`
-
+If you are using a Debian-based distro on WSL and you get a syntax error, run `export PATH="$HOME/.local/bin:$PATH"` every time you open the terminal instead.
```
Comment (fauxpark) [inline docs/newbs_getting_started.md:101]:
```suggestion
```

File: docs/newbs_getting_started.md [hunk 2]
```diff
@@ -98,7 +98,7 @@ It's possible, that you will get an error saying something like: `bash: qmk: com
 This is due to a [bug](https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=839155) Debian introduced with their Bash 4.4 release, which removed `$HOME/.local/bin` from the PATH. This bug was later fixed on Debian and Ubuntu.
 Sadly, Ubuntu reitroduced this bug and is [yet to fix it](https://bugs.launchpad.net/ubuntu/+source/bash/+bug/1588562).
 Luckily, the fix is easy. Run this as your user: `echo "PATH=$HOME/.local/bin:$PATH" >> $HOME/.bashrc && source $HOME/.bashrc`
```
Comment (fauxpark) [inline docs/newbs_getting_started.md:100]:
```suggestion
Luckily, the fix is easy. Run this as your user: `echo "PATH=$HOME/.local/bin:\"$PATH\"" >> $HOME/.bashrc && source $HOME/.bashrc`
```

**Comment by neon4272** [comment]
Yes we could
Tested on Ubuntu 20.04 WSL and it works

File: docs/newbs_getting_started.md [hunk 3]
```diff
@@ -97,8 +97,7 @@ In most situations you will want to answer Yes to all of the prompts.
 It's possible, that you will get an error saying something like: `bash: qmk: command not found`.
 This is due to a [bug](https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=839155) Debian introduced with their Bash 4.4 release, which removed `$HOME/.local/bin` from the PATH. This bug was later fixed on Debian and Ubuntu.
 Sadly, Ubuntu reitroduced this bug and is [yet to fix it](https://bugs.launchpad.net/ubuntu/+source/bash/+bug/1588562).
-Luckily, the fix is easy. Run this as your user: `echo "PATH=$HOME/.local/bin:$PATH" >> $HOME/.bashrc && source $HOME/.bashrc`
-
+Luckily, the fix is easy. Run this as your user: `echo "PATH=$HOME/.local/bin:\"$PATH\"" >> $HOME/.bashrc && source $HOME/.bashrc`
```
Comment (fauxpark) [inline docs/newbs_getting_started.md:100]:
```suggestion
Luckily, the fix is easy. Run this as your user: `echo "PATH=$HOME/.local/bin:\"$PATH\"" >> $HOME/.bashrc && source $HOME/.bashrc`

```

**Comment by zvecr** [comment]
Thanks!

**Pull request merged**